packages/shylu/shylu_dd/frosch/src/SchwarzOperators/FROSch_CoarseOperator_def.hpp

```diff
@@ -678,6 +900,23 @@ namespace FROSch {
             FROSCH_ASSERT(false,"FROSch::CoarseOperator : ERROR: Distribution type unknown.");
         }
 
+        if (OnCoarseSolveComm_) {
+            GO dimCoarseProblem = CoarseSolveMap_->getMaxAllGlobalIndex();
+            if (CoarseSolveMap_->lib()==UseEpetra || CoarseSolveMap_->getGlobalNumElements()>0) {
+                dimCoarseProblem += 1;
+            }
+            LO localVal = CoarseSolveMap_->getNodeNumElements();
+            LO sumVal;
+            LO minVal;
+            LO maxVal;
+
+            reduceAll(*CoarseSolveComm_,REDUCE_SUM,localVal,ptr(&sumVal));
```

Comment:
Shouldn't the sum over all local elements be equal to `dimCoarseProblem`?

Reply:
That was an outdated part of the code. I removed it.